packages/blockchain-wallet-v4-frontend/src/data/components/sendBtc/sagas.ts

```diff
@@ -597,7 +604,19 @@ export default ({
         )
         if (fromType === ADDRESS_TYPES.CUSTODIAL && error) {
           if (error === 'Pending withdrawal locks') {
-            yield put(actions.alerts.displayError(C.LOCKED_WITHDRAW_ERROR))
+            yield call(getWithdrawalLockCheck)
+            const rule = (yield select(
+              selectors.components.send.getWithdrawLockCheckRule
+            )).getOrFail({})
```

Comment:
make generic util saga in `packages/blockchain-wallet-v4-frontend/src/data/components/send/sagas.ts`

Reply:
call for all coins